go-controller/pkg/cni/cni.go

```diff
@@ -74,17 +74,21 @@ func (pr *PodRequest) cmdAdd() ([]byte, error) {
 	if namespace == "" || podName == "" {
 		return nil, fmt.Errorf("required CNI variable missing")
 	}
-
-	clientset, err := util.NewClientset(&config.Kubernetes)
-	if err != nil {
-		return nil, fmt.Errorf("could not create kubernetes clientset: %v", err)
+	clientSet := pr.KClient
```

Comment:
@abhat instead of doing this, just pass clientSet into cmdAdd() from HandleRequest(). And since HandleRequest() is a cniRequestFunc change the type of cniRequestFunc to take a clientSet, then pass the CNIServer's clientset into the `s.requestFunc` in `handleCNIRequest()`.